src/plugins/ga.js

```diff
@@ -1,31 +1,61 @@
 /* eslint-disable no-console */
 // From https://github.com/egoist/vue-ga/blob/master/src/index.js
-function appendScript() {
+
+function appendScript(id) {
   const script = document.createElement('script');
   script.async = true;
-  script.src = 'https://www.google-analytics.com/analytics.js';
+  script.src = 'https://www.googletagmanager.com/gtag/js?id=' + id;
   document.body.appendChild(script);
 }
 
-function init(id) {
-  appendScript();
-  window.ga =
-    window.ga ||
+// global site tag instance initialized
+function initGlobalSiteTag(id) {
+  appendScript(id);
+
+  window.dataLayer = window.dataLayer || [];
```

Comment:
Let's move the `window.dataLayer` and `window.gtag` setup *outside* of the `initGlobalSiteTag` function. This will allow users to call `gtag()` so long as it is done after the plugin's `<script>` tag:

**Plugin:**

```js
window.dataLayer = window.dataLayer || [];
window.gtag =
  window.gtag ||
  function () {
    window.dataLayer.push(arguments);
  };

$docsify.plugins = [].concat(install, $docsify.plugins);
```

**HTML:**

```html
<script src="//cdn.jsdelivr.net/npm/docsify@4/lib/docsify.min.js"></script>
<script src="//cdn.jsdelivr.net/npm/docsify@4/lib/plugins/gtag.min.js"></script>
<script>
// Advanced users can call `gtag()` on their own
window.gtag('event', 'conversion', {
  allow_custom_scripts: true,
  send_to: 'DC-ZZZZZZ/actions/locat304+standard',
});
</script>
```

Reply:
This is not work, I think we can ignore this function for the time being, and then pass the custom collector in through parameters when we need it later.